Automattic/jetpack
# Calypsoify: Update default colors to match WordPress.com

#### Changes proposed in this Pull Request:


* Update the colors used by Calypsoify to match the new default colors on WordPress.com

#### Testing instructions:





* Open the plugins screen with `?calypsoify=1`. You should see the calypsoified plugins screen
* Verify that the colors look the same as those used on WordPress.com with the default theme picked

#### Proposed changelog entry for your changes:


* Calypsoify: Update colors and color usage to match recent changes on WordPress.com
